universum/configuration_support.py

```diff
@@ -69,6 +69,8 @@ class Step:
         execution will not proceed. If no required artifacts were found in the end of the `Universum` run, it is
         also considered a failure. In case of shell-style patterns build is failed if no files or directories
         matching pattern are found.
+        Note: Universum checks both the artifact itself and an archive with the same name, because it does not know
```

Comment:
Please turn "Note" into `..note::`
Refer this page: https://universum.readthedocs.io/en/zip_artifact/configuration_support.html